### PR TITLE
update for py3

### PR DESCRIPTION
**Pull request by bauman**

### CODE REVIEW

File: schema/test_sample.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    Keyword arguments:
+    value -- dictionary to validate (required)
+    msg -- the protobuf schema to validate against (required)
+
+    Returns:
+        True: If valid input
+        False: If invalid input
+    """
+    from google.protobuf.message import EncodeError
```
Comment (Elionidas) [inline schema/test_sample.py]: your import should be at the top at the file instead of within the function

File: schema/test_sample.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    result = True
+    dict_to_protobuf.dict_to_protobuf(value, msg)
+    try:
+        msg.SerializeToString()
+    except EncodeError:
+        result = False
+    return result
+
+def filter(value, msg):
+    raise NotImplementedError("filtering implemented later")
```
Comment (Elionidas) [inline schema/test_sample.py:45]: implement pls

**Review by Elionidas — CHANGES_REQUESTED** [review]
bla bla bla

File: schema/test_sample.py (hunk 2 above)
Comment (bauman) [reply schema/test_sample.py:45]: will toss on list for next sprint

File: schema/test_sample.py (hunk 1 above)
Comment (bauman) [reply schema/test_sample.py]: changed

**Pull request merged**